recipes/spades/meta.yaml

```diff
@@ -1,29 +1,49 @@
+{% set version = "3.11.0" %}
+
 package:
   name: spades
-  version: '3.10.1'
+  version: {{ version }}
 
 source:
-  fn: SPAdes-3.10.1-Linux.tar.gz                                              # [linux]
-  url: http://spades.bioinf.spbau.ru/release3.10.1/SPAdes-3.10.1-Linux.tar.gz  # [linux]
-  md5: fb58d48a0545211aab91f43b7d17ad1b # [linux]
-  fn: SPAdes-3.10.1-Darwin.tar.gz                                             # [osx]
-  url: http://spades.bioinf.spbau.ru/release3.10.1/SPAdes-3.10.1-Darwin.tar.gz # [osx]
-  md5: f23bbad7921cff2c283e6589ccee83a2 # [osx]
+  fn: SPAdes-{{ version }}.tar.gz
+  url: http://cab.spbu.ru/files/release{{ version }}/SPAdes-{{ version }}.tar.gz
+  sha256: 308aa3e6c5fb00221a311a8d32c5e8030990356ae03002351eac10abb66bad1f
 
 build:
   number: 0
-  skip: True  # [not py2k and not py35]
 
 requirements:
   build:
+    - gcc >=4.8.2  # [not osx]
```

Comment:
is this pinning really needed?

Reply:
spades docs say it is: http://cab.spbu.ru/files/release3.11.0/manual.html#sec2.3

Reply:
This is very unusual for Bioconda, we only have gcc 4.8.5.

Reply:
pinning to >=4.8.2 is consistent with that, no?

Reply:
Yes, just saying that this is very unusual ... If you want to keep it, let's merge this PR.

Reply:
Reason to keep is, if someone later looks at the recipe as a guide for other packaging (like I do sometimes with Homebrew formulas), it's best if the requirement from spades docs is preserved.

Reply:
I agree with @notestaff - if there is a real minimum, put it in!

Reply:
I think @bgruening's concern was, this makes it look like all other bioconda recipes have _no_ minimum, which probably isn't true.